### PR TITLE
Eliminate cap on Maryland childless EITC beginning in 2023

### LINKED ISSUE #3192: Maximum MD childless EITC amount eliminated in 2023
The Comptroller of Maryland recently [announced](https://www.marylandtaxes.gov/forms/Tax_Publications/Tax_Alerts/Employer_Withholding_Tax_Alert_2023.pdf) an increase in the maximum childless Maryland EITC benefit from $530 to $600.

As pointed out by @MaxGhenis in the #3193 conversation, the above announcement is better understood in the following way:  recently passed [legislation](https://mgaleg.maryland.gov/2023RS/fnotes/bil_0007/hb0547.pdf) extends the 100% Maryland matching rate of the federal childless EITC, but **abolishes** the $530 cap on the Maryland childless EITC amount.   The $600 maximum 2023 Maryland childless EITC amount mentioned in the announcement is simply the maximum 2023 federal childless EITC amount.  That federal amount is indexed, so it will likely rise above $600 in 2024 and after.

### PR DESCRIPTION
**Pull request by martinholmer**
Fixes #3192.

### Summary
:sparkles::memo::wrench:

This pull request updates the `policyengine_us` package to include the latest Maryland childless EITC amount and source. This improves the accuracy and transparency of the state tax policy simulation.

> _`max_amount` rises from the ashes_
> _Maryland's childless EITC unleashed_
> _A minor feature for the changelog_
> _But a major blow to the elite_

### Walkthrough
*  Update the changelog entry to reflect the new feature of higher Maryland childless EITC amount in 2023 ([link](https://github.com/PolicyEngine/policyengine-us/pull/3193/files?diff=unified&w=0#diff-5885042762833a4aef51405a4036067e4e6aefa7a5091a2987d1d9d1fc2647e8R1-R4))
*  Add the new parameter value of 600 for the maximum Maryland childless EITC amount in 2023, based on the latest legislation and official guidance ([link](https://github.com/PolicyEngine/policyengine-us/pull/3193/files?diff=unified&w=0#diff-1f8484d2d4fe54afbcc9867c58752d5fb4449f57f7352c06b296f501dda45144R4), [link](https://github.com/PolicyEngine/policyengine-us/pull/3193/files?diff=unified&w=0#diff-1f8484d2d4fe54afbcc9867c58752d5fb4449f57f7352c06b296f501dda45144R12-R13))

### CODE REVIEW

File: policyengine_us/parameters/gov/states/md/tax/income/credits/eitc/childless/max_amount.yaml [hunk 1]
```diff
@@ -1,10 +1,13 @@
 description: Maryland provides up to this amount for its Earned Income Tax Credit to single childless filers.
 values:
   2019-01-01: 530
+  2023-01-01: 600
```
Comment (MaxGhenis) [inline policyengine_us/parameters/gov/states/md/tax/income/credits/eitc/childless/max_amount.yaml:4]: Per the legal code:
>(iii) For a taxable year beginning after December 31, 2019, but **before January 1, 2023,** the tax credit allowed under this paragraph may not exceed $530 for a taxable year.

And [this bill](https://mgaleg.maryland.gov/2023RS/fnotes/bil_0007/hb0547.pdf):
>Further, the $530 limit on the value of the State refundable credit for individuals without qualifying children is eliminated.

The comptroller report is just referencing the current federal EITC value for filers without qualifying children.

```suggestion
  2023-01-01: .inf
```

File: policyengine_us/parameters/gov/states/md/tax/income/credits/eitc/childless/max_amount.yaml [hunk 2]
```diff
@@ -1,10 +1,13 @@
...
 metadata:
   unit: currency-USD
   period: year
   label: Maryland childless EITC maximum
   reference:
     - title: Md. Code, Tax-Gen. § 10-704(c)(3)(iii)
       href: https://casetext.com/statute/code-of-maryland/article-tax-general/title-10-income-tax/subtitle-7-income-tax-credits/section-10-704-effective-until-6302023-for-earned-income
+    - title: Comptoller of Maryland, Requirement to Notify Employees of Potential Earned Income Tax Credit (EITC) Eligibility 2023
```
Comment (MaxGhenis) [inline policyengine_us/parameters/gov/states/md/tax/income/credits/eitc/childless/max_amount.yaml:12]: can remove since already in legal code

**Review by MaxGhenis — CHANGES_REQUESTED** [review]

File: policyengine_us/parameters/gov/states/md/tax/income/credits/eitc/childless/max_amount.yaml (hunk 1 above)
Comment (martinholmer) [reply policyengine_us/parameters/gov/states/md/tax/income/credits/eitc/childless/max_amount.yaml]: @MaxGhenis,  thanks for pointing this out.   Revisions have been made along the lines you suggested.

Comment (martinholmer) [reply policyengine_us/parameters/gov/states/md/tax/income/credits/eitc/childless/max_amount.yaml]: @MaxGhenis,  I also revised issue #3192 to reflect the correct understanding of this MD tax change.

File: policyengine_us/parameters/gov/states/md/tax/income/credits/eitc/childless/max_amount.yaml (hunk 2 above)
Comment (martinholmer) [reply policyengine_us/parameters/gov/states/md/tax/income/credits/eitc/childless/max_amount.yaml]: OK

**Pull request merged**